README.md

```diff
@@ -271,7 +265,7 @@ We need to include this function in our code. We will refer to this function in
 var advanceTask = function(task) {
   var modified = task.innerText.trim()
   for (var i = 0; i < listo.length; i++) {
-    if (listo[i].task === modified) {
+    if (listo[i].task.toUpperCase() === modified) {
```

Comment:
This was never returning true because we converted everything to uppercase before.